newsroom/templates/login.html

```diff
@@ -2,7 +2,7 @@
 
 {% block contentMain %}
 
-<div class="container-fluid py-5 overflow-auto">
+<div class="container-fluid py-5 overflow-auto nh-login-page__content">
```

Comment:
**Note:** These class additions will also need to be added to customer repos, where these templates are overridden. (such as https://github.com/superdesk/newsroom-app-cp/blob/cn-develop/server/theme/login.html#L5)

Reply:
Thanks @MarkLark86, I'll keep that in mind.